crates/common/Cargo.toml

```diff
@@ -10,15 +10,15 @@ test-utils = ["dep:metrics"]
 
 [dependencies]
 bitvec = "0.20.4"
-ethers = "1.0.0"
+ethers = "1.0.2"
```

Comment:
Do we need to make these explicit? Will semver not automate choosing these correctly?

Reply:
I've just used `cargo-update` and that tool seems to update the versions in the `Cargo.toml` files, too...